src/Illuminate/Collections/Arr.php

```diff
@@ -702,11 +702,12 @@ public static function pull(&$array, $key, $default = null)
      * Convert the array into a query string.
      *
      * @param  array  $array
+     * @param  int-mask $encoding_type (optional) PHP Query encoding type.
```

Comment:
Since we are in PHP, I'd say, we don't need to mention that
```suggestion
     * @param  int-mask $encoding_type (optional) Query encoding type.
```

Reply:
removed "PHP" from the @param in both PHPDoc comments